src/main/java/ca/serum390/godzilla/domain/sales/SalesOrder.java

```diff
@@ -0,0 +1,32 @@
+package ca.serum390.godzilla.domain.sales;
+
+import java.time.LocalDate;
+
+import org.springframework.data.annotation.Id;
+
+import org.springframework.data.relational.core.mapping.Table;
+
+import jdk.vm.ci.meta.Local;
```

Comment:
This import is causing compilation errors for me, probably also why the CI is failing